mne/time_frequency/tfr.py

```diff
@@ -1324,6 +1427,11 @@ def tfr_multitaper(inst, freqs, n_cycles, time_bandwidth=4.0,
         The decimation factor on the time axis. To reduce memory usage.
         Note than this is brute force decimation, no anti-aliasing is done.
         Defaults to 1.
+    time_range : tuple
+        Time range to which the time-frequency decomposition should be applied.
+        If time_range = (t0, t1), tfr is calculated for times in [t0, t1 - 1].
```

Comment:
Okay, conceptually speaking - isn't the TFR always *calculated* over the whole range, but the time_range param determines only what is *returned*?